### DISCUSSION TITLE
Fetching products on the basis of logged in user's location

### DISCUSSION THREAD

**Discussion by arun-esh** [arun-esh/travelx#65] [Ideas]
@jalpa015
It is not realistic idea. 

For now, what if show products as per the location of logged in user? 
We have location in the dataset that can be utilized.

**Comment by jalpa015** [comment]
@arun-esh 
Sounds good. Are we keeping track of user locations every time they access the website or based on their Sign-up location?

**Reply by arun-esh** [reply]
we need to plan accordingly as of now, NO
